docs/modules/ROOT/pages/references/architecture/control-api-user.adoc

```diff
@@ -12,8 +12,6 @@ kind: User
 metadata:
   name: bec0d928-2ae2-4cec-94a0-5f72f12b8b39 <1>
 spec:
-  displayName: Kate Demo
-  email: kate@demo.com
   defaultOrganizationRef: acme-corp
```

Comment:
If you would like to keep the resource named `User`, then I'd prefer a `spec.preferences.defaultOrganizationRef` key. Otherwise, it's not quite clear that this field is actually a preference and not a hard link/association to an organization.

Reply:
fine by me